src/main/scala/devices/spi/TLSPIFlash.scala

```diff
@@ -145,7 +145,10 @@ class TLSPIFlash(w: Int, c: SPIFlashParams)(implicit p: Parameters)
     def getOMComponents(resourceBindings: ResourceBindings, children: Seq[OMComponent] = Nil): Seq[OMComponent] = {
       Seq(
         OMSPIXIP(
-          numCS = c.csWidth,
+          instructionAddressBytes = c.insnAddrBytes,
```

Comment:
can you check an instance of OMSPIXIP and verify that it has all the parameters that SPI does as well? Its hard for me to tell from this PR how that would Just Work.

Reply:
No, it doesn't. Is that what we want? The A-spec doesn't say that we want the underlying SPI parameters as well in the Object model for SPI XIP.

Reply:
the SPIXIP `extends` SPI so it should similarly extend in the Chisel

Reply:
you mean in the object model, right?

Reply:
@mwachs5 It correctly shows the SPI params in SPIXIP now.